packages/hurl_core/src/parser/base64.rs

```diff
@@ -236,7 +236,7 @@ mod tests {
     fn test_decode_with_padding() {
         let mut reader = Reader::init("YW55IGNhcm5hbCBwbGVhcw==;");
         let decoded = parse(&mut reader);
-        assert_eq!(decoded, b"any carnal pleas");
```

Comment:
Same remark as previous, `any carnal pleas` should be left for the unit test to pass